packages/parser/test/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "isolatedDeclarations": false,
```

Comment:
Turned this off for tests because it was causing some weird typing issues in a couple fixtures.